virttest/env_process.py

```diff
@@ -1026,6 +1027,17 @@ def preprocess(test, params, env):
             env["cpu_model"] = cpu.get_qemu_best_cpu_model(params)
         params["cpu_model"] = env.get("cpu_model")
 
+    if not params.get("vcpu_driver_type"):
+        cpu_model = params.get("cpu_model")
+        if cpu_model:
+            qemu_path = utils_misc.get_qemu_binary(params)
+            for cpu_driver in utils_qemu.get_supported_devices(qemu_path, "CPU"):
+                if re.match(r'%s-\w+-cpu' % cpu_model, cpu_driver):
+                    driver_type = cpu_driver.replace("%s-" % cpu_model, "")
```

Comment:
Why do we need eliminate cpu model here ? Why not just store the complete driver name?

Reply:
`cpu_model` is order to make qemu creates with `-cpu $cpu_model`, `driver_type` is the cpu bus type